test/Microsoft.Windows.CsWin32.Tests/GeneratorTests.cs

```diff
@@ -22,6 +22,7 @@
 
 public class GeneratorTests : IDisposable, IAsyncLifetime
 {
+    private static readonly ReferenceAssemblies NetStandard20 = ReferenceAssemblies.NetStandard.NetStandard20.AddPackages(ImmutableArray.Create(new PackageIdentity("System.Memory", "4.5.4")));
     private static readonly ConcurrentDictionary<ReferenceAssemblies, Task<ImmutableArray<MetadataReference>>> TestReferencesCache = new();
```

Comment:
We should remove this, as per @sharwell pointing out this cache already exists internally. I just wasn't always benefitting from it till `NetStandard20` got cached and reused as I'm now doing in this PR.